tests/common/mod.rs

```diff
@@ -24,17 +24,11 @@ const REGION_SPLIT_TIME_LIMIT: Duration = Duration::from_secs(15);
 
 // Delete all entries in TiKV to leave a clean space for following tests.
 pub async fn clear_tikv() {
-    // DEFAULT_REGION_BACKOFF is not long enough for CI environment. So set a longer backoff.
```

Comment:
Why remove the `backoff` ? It was added as we had faced the timeout of `delete_range` in CI before.

Reply:
i removed because it seems ok on ci now

Reply:
It's ok now doesn't mean that it will always be. In my experience, the running environment of Github actions are not stable.